content/en/security/_index.md

```diff
@@ -87,24 +87,27 @@ cascade:
 
 Bring speed and scale to your production security operations. Datadog Security delivers real-time threat detection, and continuous configuration audits across applications, hosts, containers, and cloud infrastructure. Coupled with the greater Datadog observability platform, Datadog Security brings unprecedented integration between security and operations aligned to your organization's shared goals.
 
-Datadog Security includes [Application Security](#application-security), [Cloud SIEM](#cloud-siem), and [Cloud Security](#cloud-security). To learn more, check out the [30-second Product Guided Tour][14].
+Datadog Security includes [App and API Protection](#application-security), [Cloud SIEM](#cloud-siem), [Cloud Security](#cloud-security), [Code Security][20], [Workload Protection][26], and [Sensitive Data Scanner][24]. To learn more, check out the [30-second Product Guided Tour][14].
```

Comment:
```suggestion
Datadog Security includes [App and API Protection](#app-and-api-protection), [Cloud SIEM](#cloud-siem), [Cloud Security](#cloud-security), [Code Security][20], [Workload Protection][26], and [Sensitive Data Scanner][24]. To learn more, check out the [30-second Product Guided Tour][14].
```
For consistency, I think these should either all be anchor links or all external links. I'd also put these list items and the page headers in the same order, although I don't know the thinking behind either.